src/angular/codemods/sourcemaps.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import * as path from 'path';
+import * as fs from 'fs';
+
+export function addSourcemapEntryToAngularJSON(): void {
+  const angularJsonPath = path.join(process.cwd(), 'angular.json');
+
+  const angularJSONFile = fs.readFileSync(angularJsonPath, 'utf-8');
+
+  const angularJson = JSON.parse(angularJSONFile);
+
+  if (!angularJson) {
+    throw new Error('Could not find in angular.json in your project');
+  }
```

Comment:
l: If we end up here, let's provide users a fallback, which in this case is to manually enable source maps upload. 2 ideas: 

1. we link to docs
2. we show the snippet that we show in the source maps wizard if there's no pre-selected project